api/v1/dspipeline_types.go

```diff
@@ -393,6 +397,22 @@ type SecretKeyValue struct {
 	Key  string `json:"key"`
 }
 
+type ProxyConfig struct {
+	// HTTP proxy URL for outbound HTTP connections
+	// +kubebuilder:validation:Optional
+	HTTPProxy string `json:"httpProxy,omitempty"`
+	// HTTPS proxy URL for outbound HTTPS connections
+	// +kubebuilder:validation:Optional
+	HTTPSProxy string `json:"httpsProxy,omitempty"`
+	// Comma-separated list of hostnames, IP addresses, or CIDR ranges
+	// that should bypass the proxy. Recommended baseline for in-cluster traffic:
```

Comment:
Please also mention that if omitted, there is a safe default that captures these recommendations.